docs/reference/configuration/mesh.md

```diff
@@ -1,16 +1,16 @@
-<h2 id="configuration-mesh">6LoWPAN and Thread Mesh</h2>
+<h2 id="configuration-mesh">6LoWPAN, Wi-SUN and Thread Mesh</h2>
 
-This page describes build-time configurable parameters for 6LoWPAN- and Thread-based mesh networks. Mbed OS supports two main mesh protocols: 6LoWPAN-ND and Thread. Depending on the selected protocol, you can configure different set of values.
+This page describes build-time configurable parameters for 6LoWPAN-, Wi-SUN- and Thread-based mesh networks. Mbed OS supports 3 main mesh protocols: 6LoWPAN-ND, Wi-SUN and Thread. Depending on the selected protocol, you can configure different set of values.
 
 There is one mesh-capable stack in Mbed OS called Nanostack.
 
-This guide is divided in sections, and generic Nanostack configurations are first, followed by those for Thread and then 6LoWPAN-ND.
+This guide is divided in sections, and generic Nanostack configurations are first, followed by those for Thread, 6LoWPAN-ND and then Wi-SUN.
 
 For understanding the technologies and APIs, please refer to following sections before this one:
 
 - [Network connectivity in Mbed OS](../reference/networking.html) technology page.
-- [6LoWPAN Mesh technology](../reference/mesh-tech.html) page.
-- [6LoWPAN Mesh class reference](../apis/mesh-api.html) user API.
+- [Mesh technology](../reference/mesh-tech.html) page.
```

Comment:
This has been changed, now 6LoWPAN_ND, Wi-SUN and Thread are having separate page